index.bs

```diff
@@ -305,32 +305,39 @@ For the main <dfn>sanitize</dfn> operation, using a {{ParentNode}} |node|, a
          1. If "data-" is a [=code unit prefix=] of [=Attr/local name=] and
             if [=Attr/namespace=] is `null` and
             if |config|["{{SanitizerConfig/dataAttributes}}"] exists and is false:
-            1. Remove |attr| from |child|.
+            1. Then [=remove an attribute|remove=] |attr||.
       1. else if |config|["{{SanitizerConfig/removeAttributes}}"] exists and
          |config|["{{SanitizerConfig/removeAttributes}}"] [=SanitizerConfig/contains=]
          |attrName|:
-         1. Remove |attr| from |child|.
+         1. Then [=remove an attribute|remove=] |attr|.
       1. If |config|["{{SanitizerConfig/elements}}"][|elementName|] exists,
          and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/attributes}}"]
          exists, and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/attributes}}"]
          does not [=SanitizerConfig/contain=] |attrName|:
-         1. Remove |attr| from |child|.
+         1. Then [=remove an attribute|remove=] |attr|.
       1. If |config|["{{SanitizerConfig/elements}}"][|elementName|] exists,
          and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"]
          exists, and if
          |config|["{{SanitizerConfig/elements}}"][|elementName|]["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"]
          [=SanitizerConfig/contains=] |attrName|:
-         1. Remove |attr| from |child|.
-      1. If &laquo;[|elementName|, |attrName|]&raquo; matches an entry in the
-         [=navigating URL attributes list=], and if |attr|'s [=protocol=] is
-         "`javascript:`":
-         1. Then remove |attr| from |child|.
+         1. Then [=remove an attribute|remove=] |attr|.
+      1. If the [=navigating URL attributes list=] [=SanitizerConfig/contains=]
+         &laquo;[|elementName|, |attrName|]&raquo; and "`javascript:`" is a
+         [=code unit prefix=] of the result of [=strip leading and
+         trailing ASCII whitespace=] from |attr|'s [=get an attribute
+         value|value=]:
+         1. Then [=remove an attribute|remove=] |attr|.
+      1. If the [=animating URL attributes list=] [=SanitizerConfig/contains=]
+         &laquo;[|elementName|, |attrName|]&raquo; and |attr|'s
+         [=get an attribute value|value=] [=string/is=] "`href`" or
+         "`xlink:href`":
```

Comment:
We should double check that `xlink:href` is actually meaningful as it supposedly only works when `xmlns` puts `xlink` in scope which is never the case in HTML.

Reply:
`xlink:href` works in HTML regardless of namespace declarations. https://html.spec.whatwg.org/multipage/parsing.html#adjust-foreign-attributes

Reply:
@zcorpan that is `xlink:href` as an attribute, but not as an attribute value of these SVG attributes.

Reply:
Ah, I see. Still, `xmlns:xlink` is also an adjusted attribute.

Reply:
Good point, so it should work when `xmlns:xlink` is also specified. In which case we want to always ban `xlink:href` as is suggested here. (Having tests for all these combinations would be good though.)

I guess the other piece of feedback I forgot to note down here is that we should make sure we have the same processing model as these attributes have today, which regards to splitting on whitespace and such.